Resources/Prototypes/Entities/Structures/Doors/Airlocks/base_structureairlocks.yml

```diff
@@ -48,6 +48,9 @@
       path: /Audio/Machines/airlock_close.ogg
     denySound:
       path: /Audio/Machines/airlock_deny.ogg
+  - type: Tag
+    tags:
+      - RCDDeconstructWhitelist
```

Comment:
im sus on adding it to base airlock, this may be getting overriden and not properly tagged by its children as well, since the tag dicts wont merge

Reply:
How else to adress it, given airlocks do need to be deconstructed by an RCD? (Unless we don't want that)